src/db/index.js

```diff
@@ -0,0 +1,7 @@
+import { Pool } from 'pg';
+
+const pool = new Pool();
+
+const query = (text, values) => pool.query(text, values);
```

Comment:
I wonder if this line is necessary. Could we export 'pool' instead because it already has a .query method?

Reply:
It is a convenience for me, I call query instead of importing the whole pool and it's method. I don't see any other use than query at this time.